corehq/apps/accounting/migrations/0072_export_multisort_priv.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.16 on 2023-04-05 06:55
+
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.apps.accounting.models import SoftwarePlanEdition
+from corehq.privileges import EXPORT_MULTISORT
+from corehq.util.django_migrations import skip_on_fresh_install
+from django_prbac.models import Grant, Role
+
+
+@skip_on_fresh_install
+def _grandfather_export_multisort_priv(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+
+    # EXPORT_MULTISORT are Standard Plan and higher
+    skip_editions = ','.join((
+        SoftwarePlanEdition.PAUSED,
+        SoftwarePlanEdition.COMMUNITY,
+    ))
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        EXPORT_MULTISORT,
+        skip_edition=skip_editions,
+        noinput=True,
+    )
+
+
+@skip_on_fresh_install
+def _remove_export_multisort_role_and_grants(apps, schema_editor):
```

Comment:
If I'm not mistaken, this is all that need to be removed in order to revese the changes? cc @sravfeyn @kaapstorm